app/containers/Bookings/Details/OffenderDetails/index.js

```diff
@@ -117,7 +117,7 @@ const OffenderDetails = ({ offenderDetails, showPhoto }) => {
             case 'ABSENT':
               return <span>No primary address on record</span>
             case 'PRESENT':
-              return getDetails(['flat', 'street', 'town', 'postCode', 'county', 'country', 'comment'])
+              return getDetails(['flat', 'street', 'town', 'county', 'postcode', 'country', 'comment'])
```

Comment:
Since this was incorrect, would be good to have a snapshot test or similar to ensure that it remains correct in future...

Reply:
will add as a follow up